src/main/java/wooteco/subway/web/member/MemberController.java

```diff
@@ -1,46 +1,89 @@
 package wooteco.subway.web.member;
 
+import java.net.URI;
+import java.util.Set;
+
+import javax.validation.Valid;
+
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
 import wooteco.subway.domain.member.Member;
 import wooteco.subway.service.member.MemberService;
+import wooteco.subway.service.member.dto.FavoriteRequest;
+import wooteco.subway.service.member.dto.FavoriteResponse;
 import wooteco.subway.service.member.dto.MemberRequest;
 import wooteco.subway.service.member.dto.MemberResponse;
 import wooteco.subway.service.member.dto.UpdateMemberRequest;
-
-import java.net.URI;
+import wooteco.subway.web.member.interceptor.Auth;
+import wooteco.subway.web.member.interceptor.IsAuth;
 
 @RestController
+@RequestMapping(value = "/members", produces = "application/json;charset=UTF-8")
```

Comment:
```suggestion
@RequestMapping(value = "/members", produces = MediaType.APPLICATION_JSON_VALUE)
```

https://bugs.chromium.org/p/chromium/issues/detail?id=438464 를 참고해주세요
또한 가능하면 상수를 사용해주는 습관을 들여주세요!

Reply:
utf-8이 들어가지 않으면 응답값이 깨지는 현상이 발생했었습니다. 그래서 `APPLICATION_JSON_UTF8` 이걸 사용하려고 했으나, `	 * @deprecated as of 5.2 in favor of {@link #APPLICATION_JSON}` 사용하는 걸 권장하지 않아서 어쩔수 없이 하드코딩을 하게 되었습니다. 아울러 일반 JSON으로 하니깐 똑같이 깨지는 현상이 발생하더라고요. 혹시 UTF-8을 잘 표현할 수 있는 방법이 있다면 알려주세요! 찾다가 못찾았어요 ㅠㅠ

Reply:
DM으로 해결했습니다!